config/routes.rb

```diff
@@ -66,6 +66,12 @@
       get '/account_reset/confirm_delete_account' => 'account_reset/confirm_delete_account#show'
       post '/api/account_reset/send_notifications' => 'account_reset/send_notifications#update'
 
+      get '/login/two_factor/option_list' => 'two_factor_authentication/option_list#index'
```

Comment:
What about `options` instead of `option_list`? The latter feels redundant.

Reply:
I do like that name.  It was one I considered but maybe it's a bit too short.

Reply:
I thought what we've been doing is optimizing the route path names to make sense to users. For example, all our LOA3 endpoints have `verify` in the path, but internally, we use `IdV`. Similarly, a path like `/login/two_factor/options` is very clear. It means these are the options for logging in via two factor. The controller name can be completely different. We have many instances of routes where the path name is different from the controller.

Reply:
I'll change it to 'options'.  I do like that name and it's one I considered.  With respect to optimizing the paths for users I agree with that strategy: short and clear paths. I'm also a fan of having the scoping of the controllers exactly match that path so we can optimize code manageability for the developers.  Otherwise, we are introducing one level of indirection unnecessarily.